repos/system_upgrade/common/actors/opensshpermitrootlogincheck/actor.py

```diff
@@ -37,10 +46,12 @@ def process(self):
                 'Could not check openssh configuration', details={'details': 'No OpenSshConfig facts found.'}
             )
 
-        resources = [
-            reporting.RelatedResource('package', 'openssh-server'),
-            reporting.RelatedResource('file', '/etc/ssh/sshd_config')
-        ]
+        if get_source_major_version() == 7:
```

Comment:
I am afraid this will not work. "get_source_major_version" returns a string and you try to compare it against an integer.

Reply:
thanks! Fixed